### DISCUSSION TITLE
Implementing `publishTo*` for AzureDevOps?

### DISCUSSION THREAD

**Discussion by TorstenC** [docToolchain/docToolchain#1449] [Ideas]
[🇩🇪](https://github.com/docToolchain/docToolchain/discussions/1448) Wir dokumentieren unsere IT-Prozesse gerade neu und wechseln von Confluence zu AzureDevOps. Nun frage ich mich, ob wir zusätzlich zu `publishToConfluence` ein `publishToAzureDevOps` benötigen.

Einerseits nutzt AzureDevOps nur Markdown, was zu Datenverlust führen könnte. Andererseits synchronisieren wir nicht in einem Round-Trip, sodass der SPoT (Single Point of Truth) erhalten bleibt.

Zusätzliche Tasks können mit "createTask" erstellt werden. Dadurch hätte man einen eigenen Task in einem separaten Projekt, der als Skript entwickelt werden könnte. Würde Interesse bestehen, einen solchen Task später zentral zu übernehmen?

---

[🇬🇧](https://github.com/docToolchain/docToolchain/discussions/1448) We are currently transitioning our IT documentation from Confluence to AzureDevOps. Now I am considering whether we need `publishToAzureDevOps` in addition to `publishToConfluence`.

On one hand, AzureDevOps uses only Markdown, which might result in some data loss. On the other hand, as we are not synchronizing in a round-trip manner, the SPoT (Single Point of Truth) will be maintained.

Such additional tasks can also be built using "createTask." This approach would allow the task to be developed as a script within its own project. Would there be any interest in adopting such a task as a central task in the future?

**Comment by rdmueller** [comment]
Ja, es besteht Interesse einen solchen Task zu übernehmen. Ich habe schon von anderen Teams gehört, die sich auch solch einen Task gewünscht haben. Ich selbst habe mit AzureDevOps quasi nichts zu tun und kann deshalb einen solchen Task schlecht testen. Aber gemeinsam schaffen wir das sicher.

**Comment by TorstenC** [comment]
[🇩🇪](https://github.com/docToolchain/docToolchain/discussions/1448) Ich habe eine Distinct-Abfrage über `System.WorkItemType` durchgeführt und festgestellt, dass einige Typen möglicherweise individuell zu behandeln sind. Dazu zählen beispielsweise 'Doc Item', 'Requirement', 'Decision', 'Task' und andere. Ich suche nach Mitstreitern, die dieses Thema mit mir angehen und Erfahrungen aus anderen Unternehmen einbringen können. Wer ist dabei?

---
[🇬🇧](https://github.com/docToolchain/docToolchain/discussions/1448) I ran a distinct query on `System.WorkItemType` and found that some types might need to be handled individually. These include 'Doc Item,' 'Requirement,' 'Decision,' 'Task,' and others. I'm looking for collaborators who are willing to tackle this with me and share insights from other companies. Who's in?
